### PR TITLE
Add November planning session notes

### PR DESCRIPTION
**Pull request by dasanchez**
Cleaned up formatting after the meeting.

### CODE REVIEW

File: meeting_notes/20171125_planning-meeting-notes.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+- 5 GHz mesh 
+  
+- What are the immediate next steps?
+  - Toronto Reference Library:  Test deployment
+
+### Roadmap
+
+**Key milestones**
+
+1. 5GHz mesh @ Makerfest **2018 JUL** (Yurko to lead leading)
```
Comment (dcwalk) [inline meeting_notes/20171125_planning-meeting-notes.md]: lead leading :))

**Review by dcwalk — COMMENTED** [review]
I think fine, there are some markdown formatting issues that I'm happy to clean up

**Comment by dasanchez** [comment]
@dcwalk thanks, that would be great! I think the cryptpad markdown rendering behaves differently from VS Code's and Github's.

File: meeting_notes/20171125_planning-meeting-notes.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+---
+title: Meeting Notes - November 25, 2017
+location: Semaphore Studio 307, Claude T. Bissell
+attendees: 9
+date: 2017-11-25
```
Comment (garrying) [inline meeting_notes/20171125_planning-meeting-notes.md]: Should this be the date we held the meeting, or publish date of this file? Same thought about the file name

Comment (benhylau) [reply meeting_notes/20171125_planning-meeting-notes.md]: I think date of meeting

File: meeting_notes/20171125_planning-meeting-notes.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+- Met with Tony Bolla, St. Clair BIA, could connect to Kensington BIA
+- W. Queen West BIA -- connection through Adam
+- Freegeek: Meeting space @ Parkdale -- connection through dcwalk
+- Makerfestival
+- Outreach Poster! - ready to print, publish link
+- Nov 29: UofT Decentralized Tech Association talk (all welcome)
+
+5. Deployment (Pedro)
+
+- Possible locations keep growing (40+ locations), existing relationship with Rodgers
```
Comment (benhylau) [inline meeting_notes/20171125_planning-meeting-notes.md]: Who's Rodgers?

File: meeting_notes/20171125_planning-meeting-notes.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+6. Node tech (Udit)
```
Comment (benhylau) [inline meeting_notes/20171125_planning-meeting-notes.md]: No comment here?

File: meeting_notes/20171125_planning-meeting-notes.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+    - number of nodes available
+    - set dates for testing/next stages
+- **December Test Date TBA**
+  
+2. Fort York Library Outreach
+- Course curriculum development (benhylau)
+- **December delivery**
+  
+3. Campus Co-op Partnership
+- Project proposal (sindark)
```
Comment (benhylau) [inline meeting_notes/20171125_planning-meeting-notes.md]: Is sindark or @dcwalk working on this?

**Comment by benhylau** [comment]
Can we have this merged?

File: meeting_notes/20171125_planning-meeting-notes.md (hunk 2 above)
Comment (dcwalk) [reply meeting_notes/20171125_planning-meeting-notes.md]: changed.

File: meeting_notes/20171125_planning-meeting-notes.md (hunk 3 above)
Comment (dcwalk) [reply meeting_notes/20171125_planning-meeting-notes.md]: rodgers the company

File: meeting_notes/20171125_planning-meeting-notes.md (hunk 4 above)
Comment (dcwalk) [reply meeting_notes/20171125_planning-meeting-notes.md]: this was discussed above

File: meeting_notes/20171125_planning-meeting-notes.md (hunk 5 above)
Comment (dcwalk) [reply meeting_notes/20171125_planning-meeting-notes.md]: both! Added my name

**Review by dcwalk — APPROVED** [review]
Looking good, responded to all comments. We good to merge @benhylau and @dasanchez if you think so!

**Review by darkdrgn2k — APPROVED** [review]

**Review by benhylau — APPROVED** [review]

**Pull request merged**